### PR TITLE
docs(review): review docs 17/2023

### PR DESCRIPTION
**Pull request by bene2k1**
Docs review (except tutorials) week 17/2023

### CODE REVIEW

File: compute/gpu/how-to/use-gpu-with-docker.mdx [hunk 1]
```diff
@@ -121,7 +121,7 @@ The default behavior of Docker when running a container using `docker run`, is t
 
 ## How to access the GPU from the inside of a Docker container
 
-You can access the GPU of your Instance from the inside of a Docker container thanks to the preinstalled `nvidia-docker` tool. 
+You can access the GPU of your Instance from the inside of a Docker container thanks to the preinstalled `nvidia-docker` tool.
```
Comment (farhakennar) [inline compute/gpu/how-to/use-gpu-with-docker.mdx:124]:
```suggestion
You can access the GPU of your Instance from the inside of a Docker container using the preinstalled `nvidia-docker` tool.
```

File: compute/instances/api-cli/creating-backups.mdx [hunk 2]
```diff
@@ -49,7 +49,7 @@ GET https://api.scaleway.com/instance/v1/zones/<region>/images/<image_uid>
 
 An Image contains one snapshot for each volume of the instance. These snapshots are visible within the Image response, as `root_volume` and `extra_volumes` fields
 
-### Deleting a backup 
+### Deleting a backup
 
 
 To delete the image that contains a backup run the following call:
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-backups.mdx:55]:
```suggestion
 Run the following call to delete the image that contains a backup
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 3]
```diff
@@ -33,7 +33,7 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
 1. Open a terminal and type the following command, to configure the Scaleway CLI on your machine:
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:33]:
```suggestion
1. Open a terminal and type the following command to configure the Scaleway CLI on your machine:
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 4]
```diff
@@ -33,7 +33,7 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
...
 
     `scw init`
-    
+
     If you have already configured the CLI, you will see a message like the following:
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:37]:
```suggestion
    If you have already configured the CLI, you will see a message similar to the following:
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 5]
```diff
@@ -115,7 +115,7 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
     dcd8d994-b859-4e9e-aa91-2ceca41663fb  nl-ams-1  x86_64
     ff48b73a-097d-4685-b996-d3ebe50636ea  fr-par-1  x86_64
     ```
-2. Type the following command in your terminal to create an Instance: 
+2. Type the following command in your terminal to create an Instance:
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:118]:
```suggestion
2. Run the following command to create an Instance:
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 6]
```diff
@@ -168,8 +168,7 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
 
 ## Creating an Instance local volume
 
-
-1. Type the following command in your terminal to create an Instance volume with a size of 50GB: 
+1. Type the following command in your terminal to create an Instance volume with a size of 50GB:
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:171]:
```suggestion
1. Run the following command to create an Instance volume with a size of 50GB:
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 7]
```diff
@@ -188,19 +187,19 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
     State             available
     Zone              fr-par-1
     ```
-2. Write down the ID of your Instance, as you will need it in the next steps. 
+2. Write down the ID of your Instance, as you will need it in the next steps.
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:190]:
```suggestion
2. Write down the ID of your Instance. You will need it in the next steps.
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 8]
```diff
@@ -188,19 +187,19 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
...
 
-## Creating and attaching a Block Volume to an Instance 
+## Creating and attaching a block volume to an Instance
 
 <Message type="important">
   Block volumes are only available for `DEV1`, `GP1` and `RENDER` offers.
 </Message>
 
-1. Type the following command in your terminal to create a [Block Volume](/compute/instances/concepts/#block-volumes/): 
+1. Type the following command in your terminal to create a [block volume](/compute/instances/concepts/#block-volumes/):
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:198]:
```suggestion
1. Run the following command to create a [block volume](/compute/instances/concepts/#block-volumes/): 
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 9]
```diff
@@ -213,12 +212,12 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
     Project           12345678-1234-1234-4321-123456789123
     State             available
     ```
-2. Write down your Block Volume's ID as you will need it in the next steps. 
+2. Write down your block volume's ID as you will need it in the next steps.
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:215]:
```suggestion
2. Write down your block volume's ID. You will need it in the next steps.
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 10]
```diff
@@ -213,12 +212,12 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
...
 3. Type the following command in your terminal to attach your Block volume to your Instance. Make sure you replace `n4m3-0f-y0Ur-s3rv3r-id` by your server ID and `n4m3-0f-y0Ur-volume-id` by your volume ID.
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:216]:
```suggestion
3. Type the following command in your terminal to attach your block volume to your Instance. Make sure you replace `n4m3-0f-y0Ur-s3rv3r-id` by your server ID and `n4m3-0f-y0Ur-volume-id` by your volume ID.
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 11]
```diff
@@ -213,12 +212,12 @@ Scaleway Instances provide you with resources to develop, test code and deploy y
...
 
     `scw instance server attach-volume server-id=n4m3-0f-y0Ur-s3rv3r-id volume-id=n4m3-0f-y0Ur-volume-id`
 
-    You will see an output like the following: 
+    You will see an output like the following:
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:220]:
```suggestion
    You will see an output similar to the following:
```

File: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx [hunk 12]
```diff
@@ -284,11 +283,11 @@ Your Instance has been powered off.
   This action will delete all your volumes and data located on your storage server. To avoid losing data, it is recommended to perform regular snapshots.
 </Message>
 
-Ensure that your Instance is powered off, and type the following command in your terminal. Replace "your-server" with the ID of your server. 
+Ensure that your Instance is powered off, and type the following command in your terminal. Replace "your-server" with the ID of your server.
```
Comment (farhakennar) [inline compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx:286]:
```suggestion
Ensure your Instance is powered off and then run the following command, replacing "your-server" with the ID of your server.
```

File: containers/kubernetes/api-cli/cluster-monitoring.mdx [hunk 13]
```diff
@@ -45,7 +45,7 @@ Deploy the Prometheus stack in a dedicated Kubernetes [namespace](https://kubern
     [..]
     ```
 4. Verify that the created pods are all running once the stack is deployed. It is also possible to check if the 100Gi block volume was created:
```
Comment (farhakennar) [inline containers/kubernetes/api-cli/cluster-monitoring.mdx:47]:
```suggestion
4. Verify that the created pods are all running once the stack is deployed. You can also check whether the 100Gi block volume was created:
```

File: containers/kubernetes/api-cli/cluster-monitoring.mdx [hunk 14]
```diff
@@ -91,7 +91,7 @@ Deploy `Grafana` to display the `Prometheus` metrics in some pre-defined dashboa
 * `persistence.size` : 10Gi
 
 
-1. Add the Helm repo and install Grafana using `helm` with the following commands:
+1. Add the Helm repository and install Grafana using `helm` with the following commands:
```
Comment (farhakennar) [inline containers/kubernetes/api-cli/cluster-monitoring.mdx:94]:
```suggestion
1. Run the following command to add the Helm repository and install Grafana using `helm`:
```

File: containers/kubernetes/api-cli/using-load-balancer-annotations.mdx [hunk 15]
```diff
@@ -42,7 +42,7 @@ You can get a list of working annotations in the Scaleway Load Balancer [documen
 
 <Message type="note">
   - If an invalid mode is passed in the annotation, the service will output an error.
-  - If an annotation is not specified, the cloud controller manager will apply default configuration.  
+  - If an annotation is not specified, the cloud controller manager will apply default configuration.
```
Comment (farhakennar) [inline containers/kubernetes/api-cli/using-load-balancer-annotations.mdx:45]:
```suggestion
  - If an annotation is not specified, the cloud controller manager will apply the default configuration.
```

File: containers/kubernetes/api-cli/using-load-balancer-annotations.mdx [hunk 16]
```diff
@@ -96,7 +96,7 @@ This is the type of health check used.
 The default value is `tcp` and the possible values are `tcp`, `http`, `https`, `mysql`, `pgsql`, `redis` or `ldap`.
 It is possible to set the type per port, like `80:http;443,8443:https`.
```
Comment (farhakennar) [inline containers/kubernetes/api-cli/using-load-balancer-annotations.mdx:97]:
```suggestion
You can set the type per port, like `80:http;443,8443:https`.
```

File: dedibox-network/rpn/how-to/configure-rpnv1.mdx [hunk 17]
```diff
@@ -28,24 +28,24 @@ The private RPNv1 network is connected to the second physical interface of your
 
 2. Click **Create group** to create a new RPN group. The group creation interface displays.
 
-3. Enter a name for your RPN group and select the Dedibox servers you want to add to the group. Then save. 
+3. Enter a name for your RPN group and select the Dedibox servers you want to add to the group. Then save.
```
Comment (farhakennar) [inline dedibox-network/rpn/how-to/configure-rpnv1.mdx:31]:
```suggestion
3. Enter a name for your RPN group, select the Dedibox servers you want to add to the group and then save.
```

File: managed-services/iot-hub/reference-content/hubs.mdx [hunk 18]
```diff
@@ -36,7 +36,7 @@ There are three IoT Hub plans available:
   A table describing the features of each plan in detail is available in the [Hub creation wizard](https://console.scaleway.com/iot-hub/hubs/create) on the console.
 </Message>
 
-## Certificate Authority
+## Certificate authority
```
Comment (farhakennar) [inline managed-services/iot-hub/reference-content/hubs.mdx:39]: Are you sure about the lowercase? cf https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/api-cli/verify-ca-postgresql

**Review by farhakennar — APPROVED** [review]

File: containers/kubernetes/api-cli/exposing-services.mdx [hunk 19]
```diff
@@ -22,7 +22,7 @@ categories:
 
 ## Creating a Load Balancer service 
 
-1. Create a YAML manifest called `lb.yaml` and paste the following content into it. Then save the file and quit your text editor: 
+1. Create a YAML manifest called `lb.yaml` and paste the following content into it. Then save the file and quit your text editor:
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply containers/kubernetes/api-cli/exposing-services.mdx:25]:
```suggestion
1. Create a YAML manifest called `lb.yaml` and paste the following content into it. 
2. Save the file and quit your text editor:
```

File: managed-services/iot-hub/reference-content/hubs.mdx (hunk 18 above)
Comment (bene2k1) [reply managed-services/iot-hub/reference-content/hubs.mdx:39]: Both are possible - but ok
```suggestion
## Certificate Authority
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**